### ISSUE TITLE
Runtime/post-render accessibility checkup

### ISSUE THREAD

**Issue by charlie-yao** [charlie-yao/react-aria-widgets#101]
Would be good to have some kind of automated accessibility testing for when the API website is already rendered. Since Next.js is generating static files, we should be able to just test those, right?
